docs/SUMMARY.md

```diff
@@ -10,6 +10,7 @@
 		* [Supported orchestration platforms](content/release_notes/csi_rn_supported_orchestration.md)
 	* [Change log](content/release_notes/csi_rn_changelog.md)
 		* [1.6.0 (June 2021)](content/release_notes/csi_rn_changelog_1.6.0.md)
+		* [1.5.1 (July 2021)](content/release_notes/csi_rn_changelog_1.5.1.md)
```

Comment:
shouldn't this be in the [`release-1.5`](https://github.com/ibm/ibm-block-csi-driver/tree/release-1.5) branch?

Reply:
no - this is a new edition for the 1.6.0 content to include the 1.5.1 information in the changelog. 1.5.1 was already updated separately.